modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/AlignAndCalibProgress.scala

```diff
@@ -95,8 +94,8 @@ object ACProgressBar {
       val msg = if (isInError) "Error" else s.msg
       Progress(Progress.Props(
         s"Align and Calib: $msg",
-        total       = enum.all.length.toLong - 1,
-        value       = max(0, s.counter.toLong),
+        total       = enum.all.length - 1,
```

Comment:
Where does the `enum` come from. Maybe we should change the name given `enum` will be a reserved word in Dotty

Reply:
That's a `gem.util.Enumerated[AlignAndCalibStep]` defined in the same file. I'm renaming to `acSteps`.